collectors/carbonblack/cfn/collector.template

```diff
@@ -0,0 +1,108 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Alert Logic template for creating a Carbonblack log collector",
+    "Parameters": {
+        "AlertlogicAccessKeyId": {
+            "Description": "Alert Logic Access Key Id obtained from AIMS",
+            "Type": "String"
```

Comment:
This CFN template is a duplicate of the above template `collectors/carbonblack/cfn/carbonblack_collector.template`.
I suggest removing this one.

Reply:
will do, thanks